modules/ROOT/pages/alerts-on-runtime-manager.adoc

```diff
@@ -265,51 +259,55 @@ The following properties are available and can be used in the alert Email to, Su
 
 == Editing Existing Alerts
```

Comment:
should these tasks be in a separate topic from all the conceptual info above?

Reply:
Yes! But that is a story for another day.